### ISSUE TITLE
MOBILE Map Filter Panel - 1.4.4. Resize Text - Left side of panel cut off

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10141]
As a user on a mobile device, I expect all components to reflow well to be accommodated by the dimensions of a mobile screen. No components or text should be cut off or obscured.

**Linked Work Item:**
#9493 

**Expected Result:**
Content should reflow into a single column on all mobile devices, with no content obscured, cut off or requiring two dimensional scrolling to access.

**Actual Result:**
On mobile at 100% zoom, the left hand side of the map filter is cut off and not accessible at all. Problems persist and get worse at higher zoom levels.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/6079c09f-ef59-41d2-abc1-2100912775fd)

**Comment by apeters** [comment]
fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: All the elements of the map filter are now visible and accessible at 100%, but enlarging the text size still causes lots of items on this and all other components in the page to be cut off the page and text to overlap.

**Evidence:**
![Map Filter - 1 4 4 REsize Text FIXED](https://github.com/archesproject/arches/assets/77161085/5b3f3b72-986a-4c41-82e8-228e0fdf9a59)
